conformance/tests/basic/inferencepool_multiple_rules_different_pools.yaml

```diff
@@ -0,0 +1,183 @@
+---
```

Comment:
This test is part of https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/834 and should be removed here, correct?